scripts-available/CDB_Groups_API.sql

```diff
@@ -202,17 +223,17 @@ $$
     last_err = None
     while retry > 0:
       try:
-        client = SD['groups_api_client'] = httplib.HTTPConnection(params['host'], params['port'], False, params['timeout'])
+        conn = SD['groups_api_client'] = client.HTTPConnection(params['host'], params['port'], False, params['timeout'])
```

Comment:
Out of scope of the changes: I don't get how this is used. Seems to be copied from somewhere else, and I believe the original intent was to cache the connection in the [SD](https://www.postgresql.org/docs/current/plpython-sharing.html)

Reply:
You are probably right. We can open a ticket to investigate later if you want.